src/components/snackbar/index.js

```diff
@@ -0,0 +1,44 @@
+import {useState} from 'react';
+import {MdClose, MdErrorOutline} from 'react-icons/md';
```

Comment:
use open source icons instead of packaged icons

Reply:
Done!